recipes/keyutils/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.6.1" %}
+
+package:
+  name: keyutils
```

Comment:
Shouldn't we call this `libkeyutils`?

Reply:
I'm torn, it also contains `bin/keyctl`. At some point it might make sense to make this a multioutput recipe that has `libkeyutils` and `libkeyutils-static` in addition to `keyutils` but I'm not convinced it's worth splitting at the moment.

Reply:
I wasn't aware of the binary. Can you add a test for that?

Reply:
Done 👍 Unfortunately I can't do a proper test because of permissions issues in docker containers but at least it verifies the binary isn't totally broken.